zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/configuration/Pkcs12Cfg.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.impl.configuration;
+
+import java.io.File;
+import java.util.Objects;
+
+public class Pkcs12Cfg {
```

Comment:
I agree the duplication is not ideal, if there's time I wouldn't mind taking a look at a shared config module.

Reply:
We can block some time for you to take a crack at it. I honestly haven't given it that much thought, so I don't really have an idea, but open to it. That said, it shouldn't interfere with the exporter topic, so it would have to be time scoped.